src/main/java/net/fabricmc/loader/impl/discovery/RuntimeModRemapper.java

```diff
@@ -110,19 +109,19 @@ public static void remap(Collection<ModCandidateImpl> modCandidates, Path tmpDir
 				info.outputPath = outputDir.resolve(mod.getDefaultFileName());
 				Files.deleteIfExists(info.outputPath);
 
-				String accessWidener = mod.getMetadata().getAccessWidener();
+				String classTweaker = mod.getMetadata().getClassTweaker();
 
-				if (accessWidener != null) {
-					info.accessWidenerPath = accessWidener;
+				if (classTweaker != null) {
+					info.classTweakerPath = classTweaker;
 
 					try (FileSystemUtil.FileSystemDelegate jarFs = FileSystemUtil.getJarFileSystem(info.inputPath, false)) {
 						FileSystem fs = jarFs.get();
-						info.accessWidener = Files.readAllBytes(fs.getPath(accessWidener));
+						info.classTweaker = Files.readAllBytes(fs.getPath(classTweaker));
 					} catch (Throwable t) {
-						throw new RuntimeException("Error reading access widener for mod '" +mod.getId()+ "'!", t);
+						throw new RuntimeException("Error reading class tweaker for mod '" +mod.getId()+ "'!", t);
 					}
 
-					new AccessWidenerReader(mergedAccessWidener).read(info.accessWidener);
+					ClassTweakerReader.create(mergedClassTweaker).read(info.classTweaker, modNs);
```

Comment:
modNs isnt right here. I believe this is intended to be the modid, but its currently unused (loom doesnt set it).  I think this needs to be `modNs, null` or just `null`.